samples/Mvc.Client/Startup.cs

```diff
@@ -44,8 +44,8 @@ public void ConfigureServices(IServiceCollection services)
 
         .AddTwitter(options =>
         {
-            options.ClientId = Configuration["Twitter:ClientId"];
-            options.ClientSecret = Configuration["Twitter:ClientSecret"];
+            options.ClientId = Configuration["Twitter:ConsumerKey"];
+            options.ClientSecret = Configuration["Twitter:ConsumerSecret"];
```

Comment:
I think the preferred fix for this would be to change the names in appsettings.json - then the settings match the property names.